scripts/netContain/scripts/contivNet.sh

```diff
@@ -114,34 +116,39 @@ if [ "$plugin" == "kubernetes" ]; then
    fi
 fi
 
+set +eo pipefail
```

Comment:
Yes, we do expect the script to continue when the process is killed by system-tests. The only commands beyond this point is to check the status of the netplugin/netmaster process and re-spawn them as necessay (dictated by presence of /tmp/restart* files)